pennylane/templates/state_preparations/mottonen.py

```diff
@@ -235,6 +231,11 @@ def MottonenStatePreparation(state_vector, wires):
 
     This code is adapted from code written by Carsten Blank for PennyLane-Qiskit.
```

Comment:
Add a link to docs of the relevant function?

Reply:
Can't find it any more...maybe that;s why there is no link?